client/state/directly/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	initialize as initializeDirectly,
+	askQuestion as askDirectlyQuestion,
+	maximize as maximizeDirectly,
+	minimize as minimizeDirectly,
+	openAskForm as openDirectlyAskForm,
+} from 'lib/directly';
+import {
+	DIRECTLY_ASK_QUESTION,
+	DIRECTLY_INITIALIZE,
+	DIRECTLY_MAXIMIZE,
+	DIRECTLY_MINIMIZE,
+	DIRECTLY_OPEN_ASK_FORM,
+} from 'state/action-types';
+
+export const askQuestion = ( questionText, name, email ) => ( dispatch ) => {
+	dispatch( { type: DIRECTLY_ASK_QUESTION, questionText, name, email } );
+	askDirectlyQuestion( questionText, name, email );
+};
```

Comment:
Redux middleware might be more suitable for these side-effecting actions. That way the action creators can return plain objects without any unexpected effects, while the directly methods get called only when the action is dispatched.

A few examples:
 - [Automated Transfer](https://github.com/Automattic/wp-calypso/blob/master/client/state/automated-transfer/middleware.js) - clears the browser storage and refreshes the page
 - [Audio](https://github.com/Automattic/wp-calypso/blob/master/client/state/audio/middleware.js) - plays sound when an action is dispatched
 - [Notices](https://github.com/Automattic/wp-calypso/blob/master/client/state/notices/middleware.js)

cc @dmsnell